### ISSUE TITLE
Preserve URLs for historical content

### ISSUE THREAD

**Issue by ghost** [boostorg/website-v2#319]
Google search shows about 444 000 000 links to the boost website, most of them likely to be to the documentation pages.

Please, make sure that these links stay valid after the update, and the "Internet amnesia" does not happen.

The W3C has a guide on how this can be achieved: https://www.w3.org/Provider/Style/URI , although probably a lot of time has passed since, and perhaps there are better guides available.

**Comment by vinniefalco** [comment]
Yep don't worry... re-rendering all of the library documentation for all the older releases of Boost is not really practical or desirable so those will stay as-is. This will all be working correctly by the time we reach the first official release.

**Comment by vinniefalco** [comment]
> Google search shows about 444 000 000 links to the boost website

It was a mistake to have Google index the documentation for ALL old versions. This leads to situation  where for example you search for "Boost Asio" and you get 1.76.0 documentation (whoops).

We have an open issue to fix all of that and SEO-optimize things so that the search indexes produce results for the website that are very likely to be what the user wants.

**Comment by lockywolf** [comment]
I (ab)used Google as a case for links, not for general search. Finding newer versions in normal search is a great goal, it's just not what this issue is about.

I think "Google search console" should be able to tell which links might become dead after website update, although I might be mistaken.

**Comment by vinniefalco** [comment]
Well there are two issues here:

1. preserve URLs already baked into every else's content (like blog posts) and search
2. The pages on boost.org are not SEO-optimized

We are preserving the URLs, but we are also going to tweak the documentation for older versions so that they stop showing up on top of newer versions in search engines (or at all).
